Packs/CortexResponseAndRemediation/Playbooks/silent-playbook-A_user_executed_multiple_LDAP_enumeration_queries_Test.yml

```diff
@@ -1868,8 +1904,7 @@ tasks:
     skipunavailable: false
     task:
       brand: ''
-      description: Retrieves the endpoint name, agent ID and more information about
-        the IP used by the client in which the LDAP queries were executed.
+      description: Retrieves the endpoint name, agent ID and more information about the IP used by the client in which the LDAP queries were executed.
```

Comment:
Retrieves the endpoint name, agent ID, and more information about the IP used by the client for executing LDAP queries.